src/cmd/flux-wreck

```diff
@@ -123,10 +123,14 @@ prog:SubCommand {
     local id = check_jobid_arg (self, arg[1])
     local resp, err = f:rpc ("sched.cancel", { jobid = tonumber (id) })
     if not resp then
-	if err == "Function not implemented" then
+        if err == "Function not implemented" then
             prog:die ("job cancel not supported when scheduler not loaded")
-	else
-	    prog:die ("Unable to cancel %d: %s\n", id, err)
+        elseif err == "Invalid argument" then
+            prog:log ("Sending SIGKILL to %d\n", id)
+            local rc, err = f:sendevent ({signal = 9}, "wreck.%d.kill", id)
+            if not rc then self:die ("signal: %s\n", err) end
```

Comment:
Nice work!

If `sched.cancel` rpc will respond with `EINVAL` if the state of the target job is essentially not `pending` (including `running` and `completed`), hopefully there is no side effect for kill(-9) rpc.

Reply:
Yeah, I was lazy and do not check the real state of the job here. The `wreck.N.kill` event is ignored if the job isn't running.

Reply:
OK. Great. LGTM otherwise.

Reply:
Thanks for the review @dongahn!